app/konnect/dev-portal/audit-logging/replay-job.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Set up an audit log replay job
+content_type: how-to
+badge: enterprise
+---
+
+You can use the {{site.dev-portal_short_name}} Audit Logs API to configure replay jobs for [audit logging](/konnect/dev-portal/audit-logging/). 
+
+Replay jobs are useful when you have missed audit log entries due to an error or a misconfigured audit
+log webhook. You may have one replay job at a time per region, and request data from up to one week ago.
+A replay job in a region will resend data for the requested timeframe to the webhook configured for that region.
+
+## Prerequisites
+
+* [Org Admin or Portal Admin permissions](/konnect/org-management/teams-and-roles/teams-reference/)
+* Your [audit log webhook](/konnect/dev-portal/audit-logging/webhook/) must be enabled and ready to receive data. 
+
+
+## Configure a replay job
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click the Dev Portal you want to configure a replay job for.
+1. Click **Settings** in the sidebar, and then click the **Audit Logs** tab.
+1. Click the **Replay** tab.
+1. Choose a timeframe for which you want to replay the logs. 
+
+   You can choose one of the preset relative increments for up to 24 hours, or 
+   set a custom timeframe for up to 7 days.
+
+1. Apply the timeframe, then click **Send Replay**.
+
+{% endnavtab %}
+{% navtab API %}
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
```

Comment:
I don't think this needs that much description here. Delete it and point to this: https://docs.konghq.com/konnect/api/#authentication

Reply:
I just saw that you tell people to replace the PAT after the first request, I would delete this entirely and link the atuhentication doc in that sentence.